src/Sylius/Bundle/ThemeBundle/Asset/Installer/OutputAwareAssetsInstallerInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Sylius\Bundle\ThemeBundle\Asset\Installer;
+
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+interface OutputAwareAssetsInstallerInterface extends AssetsInstallerInterface
```

Comment:
I would call it OutputAwareInterface and remove the ```extends```, cause it got nothing to do with AssetsInstallerInterface.